### ISSUE TITLE
Honour/Honor not being issued

### ISSUE THREAD

**Issue by eNVy-oKay** [discovery-game/bugtracker#31]
**Describe the bug**
No type of attack generates honour/honor points

**To Reproduce**
Check honour points, attack 30 odd players, check honour points again 

**Expected behavior**
Points to be distributed as in the descriptions 

**Screenshots**
![Screenshot_20210218-135201_Discovery.jpg](https://user-images.githubusercontent.com/41645902/108426740-fddad980-71f0-11eb-888a-5a3b0e8647a5.jpg)![Screenshot_20210218-135227_Discovery.jpg](https://user-images.githubusercontent.com/41645902/108426770-08956e80-71f1-11eb-8219-ed91eff123df.jpg)![Screenshot_20210218-135317_Discovery.jpg](https://user-images.githubusercontent.com/41645902/108426781-0c28f580-71f1-11eb-939a-278c89b997e0.jpg)

**Informations (please complete the following information):**
 - Game version: Current
 - Server name: Umbriel
 - Device: Samsung
 - OS: Android
 - Version V9

**Comment by Daniel-Cooper-D02an** [comment]
Honor is not gained versus inactive players as in your screen shot, maybe we need to make sure spy reports on inactive players show this

**Comment by eNVy-oKay** [comment]
That would mean only Fanta is active on the server?

**Comment by eNVy-oKay** [comment]
The results of 74 attacks on random planets yielding no points 
![image09sm](https://user-images.githubusercontent.com/41645902/108576545-dfe5a580-72d2-11eb-8b7d-0f0825a362e8.png)
![image10sm](https://user-images.githubusercontent.com/41645902/108576542-dd834b80-72d2-11eb-80a3-3b1ee596919e.png)

**Comment by Daniel-Cooper-D02an** [comment]
Unfortunately due to the quality that the images are shown on here i cannot see the targets names very well, however from what I can see the defenders either had 0 fleet or very little and in most cases seem to be inactive (showing grey on the universe) which means in all those type of cases you will not earn honor as you have not engaged with an active players military fleet. Honor is won in battles with other active players and only against fleets with military ships in it, the amount of honor you win is dependant on the difference in power scores , the difference in fleet sizes and more importantly only when you win. I have been in battles with other active players and have won honor when I have won the battle and my opponents have won honor when I have lost the battle so have not been able to confirm this as a bug.
I will say again though that the spy reports on inactive players need to be updated to denote that no honor can be won while the player is inactive to make players aware of this.

**Comment by eNVy-oKay** [comment]
Then it is not a bug, just not enough information.

**Comment by eNVy-oKay** [comment]
![Screenshot_20210220-115241_Discovery.jpg](https://user-images.githubusercontent.com/41645902/108606954-4968c100-7372-11eb-8190-0d31440778bd.jpg)![Screenshot_20210220-115251_Discovery.jpg](https://user-images.githubusercontent.com/41645902/108606959-52599280-7372-11eb-9def-a8d5f866c4b6.jpg)

Still no points. 


![Screenshot_20210220-115522_Discovery.jpg](https://user-images.githubusercontent.com/41645902/108607000-8cc32f80-7372-11eb-8404-90fbbb3c5bd4.jpg)

**Comment by Daniel-Cooper-D02an** [comment]
I think that's down to the difference in attack fleets with yours being 30+ times bigger than his and that the player you attacked was below you in power score, both of these factors are taken into consideration when working out honor
This is taken from an announcement from Neoz back in March last year

' Tonight will be tested new Honor formulas. They will stay secret to avoid theory craft, but you can try to find them ! Not sure if it's balanced, but it's another try.
Here is was will be taken for honor calculation:
-> Fleet size compared to defencer: if you send 10 times the target ships (counted in logistic) you will have malus.
-> Power score: if you have to much PS compared to target, you will have malus.
-> Honor base is related to your power score: you can't lose/win much if you are low power scored'

Basically, honor is gained from close fights between players of the same power score areas with similar fleets, if you attack people below you with massive fleets compared to theirs you won't gain any honor

**Comment by eNVy-oKay** [comment]
So 3 levels above me and somehow a lower power score that I can not find. And a spy description that says it wi be tough to win .... 

At least I confirmed orange = Active player?

**Comment by Daniel-Cooper-D02an** [comment]
Power score is shown on the leaderboard, his is lower than yours which shows you have more ships than he does
![Screenshot_20210220-204152~2](https://user-images.githubusercontent.com/69074476/108608064-45fa2780-73bc-11eb-9093-47987c16b23f.png)
![Screenshot_20210220-204159~2](https://user-images.githubusercontent.com/69074476/108608066-48f51800-73bc-11eb-99b6-93104c0464eb.png)

All players in the universe not showing as grey are active 
Red = too powerful to attack you as you are too low below them
Orange = in the same attack range (this goes down to 0.6 your power score, even though you can attack down you will gain less to no honor to attack down) 
Green = too weak for you to attack. 
Purple = players in your own alliance

**Comment by Daniel-Cooper-D02an** [comment]
wiki page for universe view has been updated to include description for Grey players as being inactive

**Comment by eNVy-oKay** [comment]
Okay, although every target, based on the leaderboard position, is 4hrs+ away I finally found one.

![Screenshot_20210220-185741_Discovery.jpg](https://user-images.githubusercontent.com/41645902/108614341-7c30aa80-73ae-11eb-996d-e552c4bf54aa.jpg)

Colonies are to make this a less painful quest? (Searching systems one by one is not enjoyable)

**Comment by Daniel-Cooper-D02an** [comment]
A good strategy is to spread your colonies around to give you either more targets or more safety depending on your play style yes, its a very big universe so some flights will take a long time unfortunately

**Comment by eNVy-oKay** [comment]
For a starting player trying to figure out how to make things happen, those do not seem valid strategies.

The current strategy is trying to find something that does something back and see why. 

Edit: providing it can be found. The colour info was a huge help!
